### ISSUE TITLE
Prototype

### ISSUE THREAD

**Issue by m4v15** [facn2/Path18#22]
Hey guys!

Looks like you're making amazing progress on the app.

One small thing - so have you guys decided against an HTML and CSS prototype? Not that it's the end of the world, but you should definitely have some sort of prototype - it would be really good to link to the figma prototype in the readme, and maybe show your wireframes as well.

Hope it's all going well!
x

**Comment by veredrec** [comment]
Hi @m4v15!
Thank you so much for the feedback! 
After creating our Figma prototype we thought we might not need an HTML&CSS version as well, so we have decided to move on and set up the technical stuff for the project (which we did yesterday).
We agree about putting the prototype on our readme, we did it yesterday but didn't merge it yet :wink:
If you think it would be beneficial to create the HTML&CSS prototype, we can definitely do that. What are the benefits of creating both? (I didn't have the time to discuss it with the group, so it's only my opinion -
I think that since we are five people, if you recommend having both types of prototype, I don't see a problem with two of us making the prototype while other doing something else).
Thanks again!

**Comment by matthewdking** [comment]
I agree @veredrec I'm not sure we need html and css prototype! But we should definitely link to the figma prototype I will add it to the PR today

**Comment by Heathercoraje** [comment]
I agree with @matthewdking @veredrec. @m4v15  Yes we will add it to README.md

**Comment by m4v15** [comment]
OK yeah, it's fine if you guys really don't feel the need for HTML and CSS, it's a fair point of view. Mainly I just wanted to make sure you are set on the design sprint and have a prototype which then the build sprint is building towards.
